Robus/src/robus.c

```diff
@@ -162,122 +158,48 @@ error_return_t Robus_SendMsg(ll_container_t *ll_container, msg_t *msg)
     {
         msg->header.source = ctx.node.node_id;
     }
+
+    // Compute the full message size based on the header size info.
+    if (msg->header.size > MAX_DATA_MSG_SIZE)
+    {
+        data_size = MAX_DATA_MSG_SIZE;
+    }
+    else
+    {
+        data_size = msg->header.size;
```

Comment:
What about set `data_size = msg->header.size;` as default values and assign it to `MAX_DATA_MSG_SIZE` only if `msg.header.size > MAX_DATA_MSG_SIZE `.